plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/topo_diff.py

```diff
@@ -325,6 +385,43 @@ def upload_topoconfig_file(ufm_port, payload):
     else:
         return response.json, SUCCESS_CODE
 
+def get_local_cable_validation_report(ufm_port):
+    '''
+    Send request for cable validation report locally
+    '''
+    ufm_host = "127.0.0.1:{}".format(ufm_port)
+    ufm_protocol = "http"
+    resource = "/plugin/cablevalidation/cablevalidation/report/validation"
+    headers = {"X-Remote-User": "ufmsystem"}
+    response = get_request(ufm_host, ufm_protocol, resource, headers)
+    if not response:
+        return {}, ERROR_CODE
```

Comment:
same as above, print error message to the log...

Reply:
added